test/print.jl

```diff
@@ -218,8 +222,7 @@ end
       [10,Banana,:]
         [10,Banana,-1] = -10.0
         [10,Banana, 0] = 0.0
-        [10,Banana, 1] = 10.0
-    """, repl=:print)
+        [10,Banana, 1] = 10.0""", repl=:print)
```

Comment:
This was inconsistent. Some terminated by `\n` like `JuMPArray{Float64}`, some did not like `JuMPDict{Float64}`. I figured that it was more standard not to terminate with `\n`, e.g. `Array` does not terminate with `\n` so I changed `JuMPArray{Float64}` to that :)